src/components/ha-mount-picker.ts

```diff
@@ -55,9 +55,10 @@ class HaMountPicker extends LitElement {
       graphic="icon"
       .value=${__BACKUP_DATA_DISK__}
     >
-      <span
-        >${this.hass.localize("ui.components.mount-picker.use_datadisk")}</span
-      >
+      <span>
+        ${this.hass.localize("ui.components.mount-picker.use_datadisk") ||
+        "Use data disk for backup"}
```

Comment:
We can not use this translation from the supervisor panel, in this case I think its OK its not translated.
There are several other places in that panel that does it the same way.